extensions/builder.md

```diff
@@ -0,0 +1,180 @@
+# Builder Specification <!-- omit in toc -->
+
+This document specified the artifact format for a builder.
+
+A builder is an OCI image that provides a distributable build environment.
+
+A [platform][platform-spec] supporting the builder extension specification SHOULD allow users to configure the build environment with a provided builder.
+
+## Table of Contents <!-- omit in toc -->
+- [General Requirements](#general-requirements)
+  - [Builder API Version](#builder-api-version)
+  - [File/Directories](#filedirectories)
+  - [Environment Variables](#environment-variables)
+  - [Labels](#labels)
+- [Data Format](#data-format)
+  - [Labels](#labels-1)
+    - [`io.buildpacks.builder.metadata` (JSON)](#iobuildpacksbuildermetadata-json)
+    - [`io.buildpacks.buildpack.order` (JSON)](#iobuildpacksbuildpackorder-json)
+    - [`io.buildpacks.buildpack.layers` (JSON)](#iobuildpacksbuildpacklayers-json)
+    - [`io.buildpacks.lifecycle.apis` (JSON)](#iobuildpackslifecycleapis-json)
+
+## General Requirements
+A **builder** is an image that MUST contain an implementation of the lifecycle, and build-time environment, and MAY contain buildpacks. Platforms SHOULD use builders to ease the build process.
+
+### Builder API Version
+This document specifies Builder API version `0.1`.
+
+Builder API versions:
+- MUST be in form `<major>.<minor>` or `<major>`, where `<major>` is equivalent to `<major>.0`
+- When `<major>` is greater than `0` increments to `<minor>` SHALL exclusively indicate additive changes
+
+### File/Directories
+A builder MUST have the following directories/files:
+- `<CNB_BUILDPACKS_DIR>/order.toml` &rarr; As defined in the [platform specification][order-toml-spec]
+- `<CNB_BUILDPACKS_DIR>/stack.toml` &rarr; As defined in the [platform specification][stack-toml-spec]
+- `<CNB_BUILDPACKS_DIR>/lifecycle/<lifecycle binaries>` &rarr; An implementation of the lifecycle, which contains the required lifecycle binaries for [building images][lifecycle-for-build].
+
+In addition, every buildpack blob contained on a builder MUST be stored at the following file path:
+- `<CNB_BUILDPACKS_DIR>/buildpacks/...<buildpack ID>/<buildpack version>/`
```

Comment:
```suggestion
- `<CNB_BUILDPACKS_DIR>/<buildpack ID>/<buildpack version>/`
```

Reply:
Looking here: https://github.com/buildpacks/spec/blob/main/platform.md#inputs it doesn't seem the extra directory is there.